Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py

```diff
@@ -939,13 +933,14 @@ def interval_in_sec(val):
         return None
     if isinstance(val, int):
         return val
-    range_split = val.split()
-    if len(range_split) != 2:
-        raise ValueError('Interval must be "number date_range_unit", examples: (2 hours, 4 minutes,6 months, 1 day.')
-    number = int(range_split[0])
-    range_unit = range_split[1].lower()
-    if range_unit not in ['minute', 'minutes', 'hour', 'hours', 'day', 'days']:
-        raise ValueError('The unit of Interval is invalid. Must be minutes, hours or days')
+    else:
```

Comment:
This is feels more like a personal taste. Is this an enforced change?

Reply:
yeah, mypy or flake8 yelled on it.